modules/azure/billing-alert-functions/subscription-budgets/main.tf

```
@@ -0,0 +1,66 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.96.0"
+    }
+  }
+}
+
+resource "azurerm_monitor_action_group" "slack" {
+  count               = var.enable_slack == true ? 1 : 0
+  name                = "action-group-slack-${var.func_identifier}"
+  resource_group_name = var.resource_group_name
+  short_name          = "slack-ag"
+
+  webhook_receiver {
+    name = "callazurefuncapi"
+    # Using string interpolation to get full hostname of function. 
+    # slack-budget-alert comes from the package directory inside the function
+    service_uri             = "https://${var.default_hostname}/api/slack-budget-alert"
+    use_common_alert_schema = false
+  }
+}
+
+resource "azurerm_monitor_action_group" "teams" {
+  count               = var.enable_teams == true ? 1 : 0
```

This could be simplified a bit

```suggestion
  count               = var.enable_teams ? 1 : 0
```